network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -406,6 +477,7 @@ def is_current(self):
         cutoff = datetime(2020, 10, 29)
         return cutoff < review
 
+    @property
```

Comment:
we can probably lean-i-fy this, see https://github.com/mozilla/foundation.mozilla.org/issues/5668

Reply:
Yep, I agree. I assigned that ticket to me